docs/contributing.rst

```diff
@@ -18,7 +19,7 @@ Docstrings should follow `numpydoc`_ convention. We encourage extensive
 documentation.
 
 The code itself should follow `PEP8`_ convention as much as possible, with at
-most about 500 lines of code (not including docstrings) per script.
+most about 500 lines of code (not including docstrings) per file.
```

Comment:
This limit of 500 lines, plus emphasis on numpydoc convention, is the extent of the style guide. Are there any other recommendations you'd like to add regarding coding style?

Reply:
One habit I've been trying to develop for myself is to minimize the number of required positional arguments, in favor of a rich set of keyword arguments with logical default values, so that routines can be arbitrarily flexible but still easy to use.  I don't know if that's just common sense, or if it has an articulated name.